packages/app/src/app/pages/Sandbox/index.tsx

```diff
@@ -47,6 +72,7 @@ export const Sandbox: React.FC<Props> = ({ match }) => {
     actions.preferences,
     match.params,
     match.params.id,
+    query,
```

Comment:
I think this would break the collaborator invite token thingie. If we include this in the `useEffect` we will reload the sandbox page every time the URL changes, which is not intended I think. We should probably split this `useEffect` up.

This is what I did to fix it for collaborator invite tokens:

https://github.com/codesandbox/codesandbox-client/commit/9484194377b5f35229596c9883d084cc9c74d9be

Reply:
That makes sense, maybe the best idea is to disable eslint for those two effects because I don't want that in the effect I made too